crates/node/core/src/args/engine.rs

```diff
@@ -63,6 +63,10 @@ pub struct EngineArgs {
     /// Enable precompile cache
     #[arg(long = "engine.precompile-cache", default_value = "false")]
     pub precompile_cache_enabled: bool,
+
+    /// Enable state root fallback testing
+    #[arg(long = "engine.test-state-root-fallback", default_value = "false")]
+    pub test_state_root_fallback: bool,
```

Comment:
I would just name the arg here `state-root-fallback` or `fallback-state-root`, but make sure the comment describes why it would be used / that it's for testing

Reply:
makes sense, done